CMakePresets.json

```diff
@@ -219,7 +263,7 @@
       "displayName": "Build",
       "description": "Build the project in debug",
       "targets": [
-          "all"
+        "all"
```

Comment:
You modified the cmake configure presets, but not the build or test presets. That means that `cmake --build --preset windows_ninja` 'might work', but it's inherently misleading due to the hidden nature of windows_ninja configure preset.

... is that fine? I think that's fine, the only difference between the two is a configure flag.

I'll add a test preset when I mess around with CI yml files.

Reply:
nope jk it's borked

```
> cmake --build --preset windows_ninja
CMake Error: Cannot use hidden configure preset in C:/Users/Andrew/source/repos/ladybird: "windows_ninja"
Available configure presets:

  "windows_ci_ninja"    - Windows Continuous Integration (ClangCL and Ninja)
  "windows_dev_ninja"   - Window Development (ClangCL and Ninja)
  "windows_dev_msbuild" - Window Development (ClangCL and MSBuild)
  ```

Reply:
... I'll just push my fixes in a new PR